### ISSUE TITLE
The capacity map on explorer.threefold.io needs to be improved

### ISSUE THREAD

**Issue by samtaggart** [threefoldtech/nodes-explorer#10]
<img width="493" alt="Screen Shot 2021-08-06 at 9 27 21 AM" src="https://user-images.githubusercontent.com/50134479/128466032-41c52240-59b5-4271-b736-7cd3f1a03a31.png">

The capacity map on explorer.threefold.io is a very bad user experience. Some recommendations:

- [ ] The user should be able to zoom into the map (this one is crucial)
- [ ] The user should be able to see roughly where the nodes are located within the country
- [ ] The user should be able to see how much capacity is available in the country

I don't know where this falls in priority but please get in touch with me if / when you have time @DylanVerstraete to address this and let's see what can be done. I think the zoom functionality is really crucial.

**Comment by DylanVerstraete** [comment]
In order to have a zoom on the map we need to use another library, since there is no time allocated to work on this we will have to plan this later this year.

**Comment by scottyeager** [comment]
I created a prototype for based on [Leaflet](leafletjs.com) that fulfills the first two items. See attached HTML file for a static example, generated from the included Python script. It pulls the geolocation data for nodes from the Grid 2 explorer. It looks like the Grid 3 explorer provides the same and I think it should be straightforward to integrate in both cases.

Notes:
* Clustering is provided by the [markercluster](https://github.com/Leaflet/Leaflet.markercluster) plugin. I found this to be necessary for performance and also gives a better sense for node density. `maxClusterRadius` can be tuned to adjust the cluster granularity, and there are a few other options to tweak clustering behavior which are documented on the repo.
* My example uses Ersi/ArcGIS map tiles, which they serve for free with no API key required. Apparently there's a two million tile per month [limit](https://developers.arcgis.com/pricing/), but I'm not sure how this is metered or if it's enforced. Options for free tile providers are limited, and this is the best I found. The entire (long) attribution must be included intact.
* It would be awesome to include more information about which farms are which, what kind of capacity the nodes have, etc. Formatting node data into GeoJSON and using capabilities in Leaflet to display marker data on click would be a logical next step.

[node-map.zip](https://github.com/threefoldtech/nodes-explorer/files/7708805/node-map.zip)
